config/initializers/mcb_audit.rb

```diff
@@ -0,0 +1,10 @@
+if ENV.key?('MCB_AUDITED_USER_EMAIL')
```

Comment:
So do we supply the email here or not? If not, we can give this a more appropriate name.

Reply:
oops - fixed